### DISCUSSION TITLE
Is it possible to do conditions within a goreleaser.yml?

### DISCUSSION THREAD

**Discussion by dwburke** [goreleaser/goreleaser#2132] [Q&A] [answered]
I need to build a project rpm without a dependency on dev servers, but when I build it for staging or production I need that dep in place.  Is there a way within goreleaser to do this?  Something along the lines of a template condition or something else?
i.e.
```
{{ if .Env.Dev }}
- rpm-name
{{end}
```

Or maybe there's a better way that doesn't include maintaining two separate yaml's?  which is the direction I'm afraid I'll be forced to use atm.

**Comment by caarlos0** [comment, marked as answer]
hey, some fields are templateable, but I think dependencies aren't. You're probably better of with multiple goreleaser configs or using `envsubst` or something like that before goreleaser...
